app/templates/index.hbs

```diff
@@ -0,0 +1,11 @@
+<h1>remEMBER</h1>
```

Comment:
Similar to the comments above, this should live in a `reminders.hbs` file. When you generate the route with the CLI that file will be automatically generated.